vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -519,6 +521,30 @@ private void validateSelectionEnabledState(PropertyChangeEvent event) {
         }
     }
 
+    /**
+     * Compares two value instances to each other to determine whether they are
+     * equal. Equality is used to determine whether to update internal state and
+     * fire an event when {@link #setValue(Object)} or
+     * {@link #setModelValue(Object, boolean)} is called. Subclasses can
+     * override this method to define an alternative comparison method instead
+     * of {@link Objects#equals(Object)}.
+     *
+     * @param value1
+     *            the first instance
+     * @param value2
+     *            the second instance
+     * @return <code>true</code> if the instances are equal; otherwise
+     *         <code>false</code>
+     */
+    @Override
+    protected boolean valueEquals(T value1, T value2) {
+        if (value1 == null && value2 == null)
+            return true;
+        if (value1 == null || value2 == null)
+            return false;
+        return getItemId(value1).equals(getItemId(value2));
```

Comment:
Don't you have the Vaadin Java coding conventions set up in the IDE ? Missing brackets ...

Reply:
Actually, I do have that setup in my IDE (IntelliJ) but it is complaining about one-liner `if` statements to have the bracket. Although, personally I am a fan of having brackets always for one-line `if` and `for` statements, but I couldn't find it anywhere in the following places: 

- https://sites.google.com/a/vaadin.com/fw-flow-team-s-wiki/contributing-code/flow-way-of-devloping 
- https://sites.google.com/a/vaadin.com/fw-flow-team-s-wiki/contributing-code/code-style

Reply:
Done.